index.html

```diff
@@ -1,20 +1,2959 @@
+                                          <div class="txtsubs col-12 col-md-7 text-center pt-2 mx-0 px-0">
+                                              <h3>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h3>
+                                              <p>
+                                                  All Access + Weekly Delivery.
+                                                  <br>
+                                                  Choose A Membership That's Perfect for You!
+                                              </p>
+                                              <button class="btnsubs" type="button">SUBSCRIBE</button>
+                                          </div> <!--.txtsubs-->
+                                      </div>
+                                  </div>
+                              </div>
+                            </section><!--.subscon-->
+                          </div> 
+                          <div class="col-12 col-lg-5 mx-0 px-0">
+                            <section class="newscon py-5 flex-md-column">
+                                <div class="substitle">Newsletter</div>
+                                <div class="newsmain p-4">
+                                    <div class="imgnews mx-0 px-0">
+                                        <div class="actimgnews" style="background: url(&quot;https://d.newsweek.com/en/full/1541302/starting5-homepage.png?w=397&amp;h=265&amp;q=90&amp;f=742bce9a6f72dd8f19ba2e7fd5db06e5&quot;) center center / cover no-repeat;"></div>
+                                        <!-- <img class="actimgnews" src="https://d.newsweek.com/en/full/1541302/starting5-homepage.png?w=397&amp;h=265&amp;q=90&amp;f=742bce9a6f72dd8f19ba2e7fd5db06e5" alt="hand holding phone"> -->
+                                    </div>
+                                    <div class="txtsubs p-3 text-center">
+                                        <h3 class="text-uppercase">The Starting 5</h3>
+                                        <p>See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+                                        <form class="free-signup d-flex">
+                                            <input type="text" class="form-text hformnews" id="sub-email" placeholder="Email address">
+                                            <input type="submit" class="form-submit hformnews" value="Free Sign Up">
+                                        </form>
+                                    </div> <!--.txtsubs -->
+                                </div>
+                            </section> <!--.newscon-->
+                        </div> 
+                  </div>
+
+            </div><!--.sbuscribe-block-->
+        </div><!--.page-bottom .container-fluid-->
+        <!-- footer section -->
+        <footer class="container-fluid main-footer mt-3 border-top pt-3">
+        <div class="row d-flex align-items-end">
```

Comment:
- [ ] Fix the spacing of the line under the first row of the footer.

#### Your footer:
![Screenshot_2021-02-05 Newsweek clone(1)](https://user-images.githubusercontent.com/16193212/107027408-ddd7fe80-678a-11eb-964f-4b4ad8b529f8.png)



#### Original footer:
![footer_tablet](https://user-images.githubusercontent.com/16193212/107027459-ec261a80-678a-11eb-95a9-d845886adbcb.png)